test/calculator_test.cc

```diff
@@ -0,0 +1,74 @@
+#include "calculator.h"
+
+#include <gtest/gtest.h>
+
+namespace dev {
+namespace testing {
+
+class CalculatorTest : public ::testing::Test {
+ public:
+  void SetUp() override {}
+  void TearDown() override {}
+  dev::Calculator calculator_;
+};
+
+// 1. Basic operations
+
+TEST_F(CalculatorTest, CalcAdd) {
+  ASSERT_EQ(3.0, calculator_.calc("1 + 2"));
+}
+
+TEST_F(CalculatorTest, CalcSub) {
+  ASSERT_EQ(-1.0, calculator_.calc("1 - 2"));
```

Comment:
We are expecting calculator in polish notation https://en.wikipedia.org/wiki/Polish_notation

Reply:
updated.